### PR TITLE
remove deprecated CompoundComparator getsetname

### PR DESCRIPTION
**Pull request by ijusti**

### CODE REVIEW

File: src/main/java/org/springframework/data/aerospike/core/BaseAerospikeTemplate.java [hunk 1]
```diff
@@ -215,10 +209,14 @@ WritePolicy ignoreGenerationDeletePolicy() {
                 .build();
     }
 
+    protected String getSetName(Object id, AerospikePersistentEntity<?> entity) {
```
Comment (Aloren) [inline src/main/java/org/springframework/data/aerospike/core/BaseAerospikeTemplate.java:212]: why this change is needed?

Comment (ijusti) [reply src/main/java/org/springframework/data/aerospike/core/BaseAerospikeTemplate.java:212]: I will override this method for my purposes

Comment (Aloren) [reply src/main/java/org/springframework/data/aerospike/core/BaseAerospikeTemplate.java:213]: This is not the only place where set name is requested and moreover it may likely break in future releases as it is not a public API.
